openlibrary/macros/EnrollUsergroup.html

```diff
@@ -0,0 +1,12 @@
+$def with (usergroup, endpoint, callback="")
```

Comment:
This file doesn't look like it's used anywhere? Or am I missing something?

Reply:
It's used on https://openlibrary.org/sponsorship as a macro